apps/payment/appconfig.py

```diff
@@ -7,3 +7,6 @@ class PaymentConfig(AppConfig):
 
     def ready(self):
         super(PaymentConfig, self).ready()
+        # The following stops pycharm from nagging about unused import statement
+        # noinspection PyUnresolvedReferences
+        import apps.payment.signals  # noqa: F401
```

Comment:
OK, so I have a more substantial comment this time.

From reading the docs, it seems like this `ready()` method can end up being called multiple times. When testing locally it seems to actually be the case as well, and it seems to cause the signal receivers to actually be registered twice as well. The receipt also seemingly gets sent twice.

Do you know anything about this behavior? Can we make this import only run once, somehow?

Reply:
I have no idea about it. I just did it as all the other apps did it

Reply:
Sorry, I can't read. The import might run twice but the listeners only run once.